lexers/hare.go

```diff
@@ -0,0 +1,60 @@
+package lexers
```

Comment:
Please convert this to an XML lexer, I'm no longer accepting Go-based lexers unless there's a good reason. Conversion is fairly mechanical, see other lexers in lexers/embedded for examples.

Reply:
Done.

Just a couple suggestions regarding XML lexer format errors: I had spent some time trying to debug why the embedded lexer wasn't working for any Hare code I tried:
```console
$ chroma --json x.ha 
[
  {"type":"Error","value":"//test\nlet x = 1;\n"}
]

```

Until I figured out that I was supposed to wrap the `<include>` elements in `<rule>` elements; it would be great if Chroma printed out a clear error for this case, with a specific line and character or range. After that, I tried putting two `<include>` elements in one rule, and I got another somewhat cryptic error (which was fixed by putting each `<include>` in a separate rule):
```console
$ chroma --json x.ha
chroma: error: Hare: failed to compile rules: embedded/hare.xml: duplicate mutator
```

This is better than before, but it would be great if the error was more specific and it pointed to a specific line and character or range as mentioned already.